exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -452,8 +452,67 @@ func (c *CoinbasePro) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse,
 
 // GetOrderInfo returns information on a current open order
 func (c *CoinbasePro) GetOrderInfo(orderID string) (order.Detail, error) {
-	var orderDetail order.Detail
-	return orderDetail, common.ErrNotYetImplemented
+	genOrderDetail, errGo := c.GetOrder(orderID)
+	if errGo != nil {
+		return order.Detail{}, fmt.Errorf("error retrieving order %s : %s", orderID, errGo)
+	}
+	od, errOd := time.Parse(time.RFC3339, genOrderDetail.DoneAt)
+	if errOd != nil {
+		return order.Detail{}, fmt.Errorf("error parsing order done at time: %s", errOd)
+	}
+	os, errOs := order.StringToOrderStatus(genOrderDetail.Status)
+	if errOs != nil {
+		return order.Detail{}, fmt.Errorf("error parsing order status: %s", errOs)
+	}
+	tt, errOt := order.StringToOrderType(genOrderDetail.Type)
+	if errOt != nil {
+		return order.Detail{}, fmt.Errorf("error parsing order type: %s", errOt)
+	}
+	ss, errOss := order.StringToOrderSide(genOrderDetail.Side)
+	if errOss != nil {
+		return order.Detail{}, fmt.Errorf("error parsing order type: %s", errOss)
+	}
+	response := order.Detail{
+		Exchange:        c.GetName(),
+		AccountID:       "",
+		ID:              genOrderDetail.ID,
+		CurrencyPair:    currency.NewPairDelimiter(genOrderDetail.ProductID, "-"),
+		OrderSide:       ss,
+		OrderType:       tt,
+		OrderDate:       od,
+		Status:          os,
+		Price:           genOrderDetail.Price,
+		Amount:          genOrderDetail.SpecifiedFunds,
+		ExecutedAmount:  genOrderDetail.FilledSize,
+		RemainingAmount: genOrderDetail.Size - genOrderDetail.ExecutedValue,
+		Fee:             genOrderDetail.FillFees,
+		Trades:          nil,
```

Comment:
Both `Trades` and `AccountID` don't need to be set here as they are the default values

Reply:
Hey, thanks for reviewing. Did you actually mean **don't**? Sorry, I don't understand the ask.

Reply:
You're setting `Trades` and `AccountID` to their default values, so you don't need to have them there. ie
```
order.Detail{
		Exchange:        c.GetName(),
		ID:              genOrderDetail.ID,
		CurrencyPair:    currency.NewPairDelimiter(genOrderDetail.ProductID, "-"),
		OrderSide:       ss,
		OrderType:       tt,
		OrderDate:       od,
		Status:          os,
		Price:           genOrderDetail.Price,
		Amount:          genOrderDetail.SpecifiedFunds,
		ExecutedAmount:  genOrderDetail.FilledSize,
		RemainingAmount: genOrderDetail.Size - genOrderDetail.ExecutedValue,
                Fee:             genOrderDetail.FillFees,
}
```
Is the same thing as what you have written. So you don't need those lines

Reply:
got it now, cheers